app/plan.go

```diff
@@ -144,3 +145,28 @@ func PlanRemove(planName string) error {
 	}
 	return err
 }
+
+type Router struct {
+	Name string `json:"name"`
+	Type string `json:"type"`
+}
+
+func RoutersList() ([]Router, error) {
```

Comment:
I don't think that this function belongs to this package. It should be in the package "router".